lib/showoff.rb

```diff
@@ -1375,13 +1375,16 @@ def valid_cookie
   end
 
   get %r{(?:image|file)/(.*)} do
-    path = params[:captures].first
-    full_path = File.join(settings.pres_dir, path)
-    if File.exist?(full_path)
-        send_file full_path
-    else
-        raise Sinatra::NotFound
-    end
+	path = params[:captures].first
+	full_path = File.join(settings.pres_dir, path)
+
+	if File.exist?(full_path)
+		send_file full_path
+	elsif File.exist?(path) # windows compatibility
```

Comment:
Is `.pres_dir` not working properly for you? I'll spin up a windows vm in a bit and validate that.